go/webhook/testcases/testdata/testcases/auth0/empty_batch.json

```diff
@@ -11,11 +11,10 @@
   },
   "output": {
     "response": {
-      "status": 200,
-      "body": "OK"
+      "status": 400,
```

Comment:
why change in status ? Is there any corresponding change done for this ?

Reply:
The integration test was failing to validate status. Hence had to make this change.
> Is there any corresponding change done for this ?

On server side no changes are done.